### PR TITLE
Remove unsafe instances from MutList

### PR DESCRIPTION
**Pull request by ninaandrup**
#2642

### CODE REVIEW

**Comment by magnus-madsen** [comment]
OK. I realize now that my description of the problem was not exact. While I DO want to remove the Functor instance, I still want to retain most functions.

File: main/src/library/MutList.flix [hunk 1]
```diff
@@ -108,14 +82,6 @@ namespace MutList {
     ///
     pub def memberOf(x: a, v: MutList[a]): Bool & Impure with Eq[a] = exists(y -> y == x, v)
 
-    ///
-    /// Optionally finds the smallest element of `v` according to the `Order` on `a`.
-    ///
-    /// Returns `None` if `v` is empty.
-    ///
-    pub def minimum(v: MutList[a]): Option[a] & Impure with Order[a] =
```
Comment (magnus-madsen) [inline main/src/library/MutList.flix:116]: For example, even though the functor instance is removed, we should still have a minimum function.

**Comment by magnus-madsen** [comment]
In other words, when you remove say `Functor` if something then breaks because of missing function X then I think X should be implemented inside the Array namespace. Sorry for not being more clear.

File: main/src/library/MutList.flix (hunk 1 above)
Comment (ninaandrup) [reply main/src/library/MutList.flix:116]: I accidentally merged with the other PR branch #2704, meaning it deleted these functions again, and it won't let me revert...

**Comment by ninaandrup** [comment]
> OK. I realize now that my description of the problem was not exact. While I DO want to remove the Functor instance, I still want to retain most functions.

More like this right? #2707

**Comment by magnus-madsen** [comment]
> > OK. I realize now that my description of the problem was not exact. While I DO want to remove the Functor instance, I still want to retain most functions.
> 
> More like this right? #2707

Yes

**Pull request closed without merging**